### ISSUE TITLE
Write alt text for images, logos, buttons

### ISSUE THREAD

**Issue by srhhnry** [cal-itp/benefits#1392]
In an effort to improve accessibility for the site, all images, buttons, and logos will have alt text

- [x] Research best practices for alt text
- [x] Write alt text for buttons
- [x] Write alt text for images
- [x] Write alt text for non-decorative graphics/logos
- [x] Find a home for alt text in Figma
- [x] Discuss with dev
- [x] #1410

**Comment by indexing** [comment]
Do we need to translate alt text copy to Spanish so it's available when a person changes languages?

**Comment by srhhnry** [comment]
Good catch--yes it does appear that alt text should be translated as well (and that makes good sense). Added it to the check list, though we might want to make a different ticket for that. Will leave that up to the devs.

**Comment by srhhnry** [comment]
That sounds great @indexing. Do you mind accounting for that in your process and then I'll go ahead and close this ticket?

**Comment by indexing** [comment]
I split the translation effort into a new issue. During planning, we'll decide when to address it @srhhnry .
